cabal-install/cabal-install.cabal

```diff
@@ -333,6 +333,7 @@ executable cabal
         zlib       >= 0.5.3    && < 0.7,
         hackage-security >= 0.5.2.2 && < 0.6,
         text       >= 1.2.3    && < 1.3,
+        zip-archive >= 0.3.2.5 && < 0.4,
```

Comment:
you actually need to drop this one too!

...otherwise CI complains about the divergence (as it does right now):

```
$ ./Cabal/misc/travis-diff-files.sh
+git status
+git diff-files -p --exit-code
diff --git a/cabal-install/cabal-install.cabal b/cabal-install/cabal-install.cabal
index 985cc2e..51d4335 100644
--- a/cabal-install/cabal-install.cabal
+++ b/cabal-install/cabal-install.cabal
@@ -333,7 +333,6 @@ executable cabal
         zlib       >= 0.5.3    && < 0.7,
         hackage-security >= 0.5.2.2 && < 0.6,
         text       >= 1.2.3    && < 1.3,
-        zip-archive >= 0.3.2.5 && < 0.4,
         parsec     >= 3.1.13.0 && < 3.2
 
     if flag(native-dns)
The command "./travis-${SCRIPT}.sh -j2" exited with 1.
```